wagtail_airtable/mixins.py

```diff
@@ -362,3 +364,29 @@ def delete(self, *args, **kwargs):
 
     class Meta:
         abstract = True
+
+
+class ImportButton(Button):
+    template_name = "wagtail_airtable/_import_button.html"
+
+    def get_context_data(self, parent_context):
+        context = super().get_context_data(parent_context)
+        context["csrf_token"] = parent_context["csrf_token"]
+        context["model_opts"] = parent_context["model_opts"]
+        context["next"] = parent_context["request"].path
```

Comment:
note to self: this is intended to be used in the Snippets context which build on the generic views and populate these via the components context